scripts/exglobal_prep_snow_obs.py

```diff
@@ -21,5 +21,6 @@
     # Instantiate the snow prepare task
     SnowAnl = SnowAnalysis(config)
     SnowAnl.prepare_GTS()
-    if f"{ SnowAnl.task_config.cyc }" == '18':
+    SnowAnl.task_config.cyc = f'{SnowAnl.task_config.cyc:02d}'
```

Comment:
Why is this needed?

Reply:
To keep the leading '0' for the 00z cycle to be '00', so that the following `if f"{ SnowAnl.task_config.cyc }" == '00':` is true for the 00z

Reply:
can't it just be `if SnowAnl.task_config.cyc == 0`?

Reply:
Made the according changes. Thanks.